packages/lesswrong/components/Layout.jsx

```diff
@@ -34,7 +34,7 @@ const Layout = ({currentUser, children, currentRoute, params, client}, { userAge
       <MuiThemeProvider muiTheme={customizeTheme(currentRoute, userAgent, params, client.store)}>
         <div>
           <Helmet>
-            <title>LessWrong 2.0</title>
+            <title>{getSetting('AlignmentForum', false) ? 'Alignment Forum' : 'LessWrong 2.0'}</title>
```

Comment:
Since we're gearing p for EA forum anyway, does it make more sense for this to be an overall "forumTitle" setting rather than Alignment Forum specific thing?

Reply:
(I think most of the other AF setting stuff probably warrants an actual boolean flag rather than a string, this particular one just seemed weird)